config/consensus.go

```diff
@@ -574,19 +574,42 @@ type MiningRules struct {
 	// can suspend for not proposing "lately" (In 10x expected interval, or
 	// within a grace period from being challenged)
 	MaxMarkAbsent int
+
+	// Challenges occur once every challengeInterval rounds.
+	ChallengeInterval uint64
+	// Suspensions happen between 1 and 2 grace periods after a challenge. Must
+	// be less than half MaxTxnLife to ensure the Block header will be cached
+	// and less than half ChallengeInterval to avoid overlapping challenges. A larger
+	// grace period means larger stake nodes will probably propose before they
+	// need to consider an active heartbeat.
+	ChallengeGracePeriod uint64
+	// An account is challenged if the first challengeBits match the start of
+	// the account address. An online account will be challenged about once
+	// every interval*2^bits rounds.
+	ChallengeBits int
 }
 
 // miningRules should be extended, never changed, since old blocks must retain
 // their behavior.
 var miningRules = [...]MiningRules{
-	{Enabled: false},
 	{
-		Enabled:       true,
-		Percent:       75,
-		GoOnlineFee:   2_000_000,           // 2 algos
-		MinBalance:    100_000_000_000,     // 100,000 algos
-		MaxBalance:    100_000_000_000_000, // 100M algos
-		MaxMarkAbsent: 32,
+		Enabled: false,
+		// Because the eligibility check has a lookback, we need these set even
+		// here before mining begins.
+		MinBalance: 30_000_000_000,     // 30,000 algos
+		MaxBalance: 50_000_000_000_000, // 50M algos
```

Comment:
@jannotti sanity checking this max number...

Reply:
You tell me.

Reply:
resolved. 70M